spock-container-test-app/build.gradle

```diff
@@ -1,67 +1,66 @@
-plugins {
-        id "groovy"
-        id "org.grails.grails-web"
-        id "org.grails.grails-gsp"
-        id "war"
-        id "idea"
-        id "com.bertramlabs.asset-pipeline"
-        id "eclipse"
+buildscript {
+    repositories {
+        maven { url = 'https://repo.grails.org/grails/core/' }
+        mavenCentral()
+    }
+    dependencies {
+        classpath "com.bertramlabs.plugins:asset-pipeline-gradle:$assetPipelineVersion"
```

Comment:
Does this plugin really need the asset plugin?

Reply:
This is not the build file of the plugin, but of the functional tests project.